docs/src/main/asciidoc/validation.adoc

```diff
@@ -397,6 +397,8 @@ provided the supported locales have been properly specified in the `application.
 quarkus.locales=en-US,es-ES,fr-FR
 ----
 
+Alternatively, you can use `all` to make native-image executable to include all available locales. It inflates the executable size though.
```

Comment:
If I am not mistaken the added size has a fixed upper limit, right? Do you know how much that is? E.g. what's the difference between not including locales and including all locales? I guess providing this number would make it much easier for someone to take the decision (without having to try on their own).

Reply:
The difference between having just e.g. 2 locales, English and Czech
and having *all* locales is easy more than 23 MB.  I will add that detail in the text.